app/code/Magento/PageCache/Model/App/Request/Http/IdentifierForSave.php

```diff
@@ -1,16 +1,26 @@
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Copyright 2024 Adobe
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains
+ * the property of Adobe and its suppliers, if any. The intellectual
+ * and technical concepts contained herein are proprietary to Adobe
+ * and its suppliers and are protected by all applicable intellectual
+ * property laws, including trade secret and copyright laws.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained from
+ * Adobe.
```

Comment:
This copyright header is too big.
Also the year should be the year the file was **created**, not when it was last updated.

So in this case the copyright header should be:
```
 * Copyright 2023 Adobe
 * All Rights Reserved.
```

Nothing else is needed. Not sure where you got that other paragraph from?

Same remark goes for the other files.

Reply:
It's better to be moved to separate file  COPYING.txt or LICENSE.txt for better opportunity for update the policies. 

It's takes years to remove @author tag (comments block) from files, what will happens if all files requires updated license ...

Reply:
Please do not use any year dates, just use the default magento header here:

```
<?php
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
```

Reply:
new rule from magento coding standard (https://github.com/magento/magento-coding-standard/blob/develop/Magento2Framework/Sniffs/Header/CopyrightValidation.php)
I'm  copy from core
Have 94 file same format

Reply:
It couldn't have been made any longer *lol* but thanks, I hadn't thought of that either

Reply:
So I discussed this with somebody from Adobe a last week on Slack, he said to me that the new requirement from the Adobe legal team (which came after a lot of discussions), should be:

```
Copyright [year file created] Adobe
All Rights Reserved.
```

I totally agree that it's silly to change these copyright headers yet again, it has been changing way too often in the last 8 years or so. And I would love if we could just put it in an external file and reference to that and never have to change it again in the future, but the Adobe legal team disagrees with this approach for some unknown reason.

I also argued on Slack that the check in coding standards is too lose, as it only checks for the first line and you can basically put whatever you like after the first line, they agreed and they will fix this in a new version of coding standards someday in the future so it's clearer what is required.

So for now, can you please update it to only those 2 lines and use git to figure out the year each file was created and use that year. Thanks :)